test/test_overrides.py

```diff
@@ -838,5 +838,27 @@ def test_newones(self):
         n = t.new_ones((1, 2))
         self.assertEqual(type(n), SubTensor2)
 
+class TestBroadcastAllOverride(TestCase):
+    " test for gh-37141
```

Comment:
```suggestion
    """test fo rgh-37141"""
```

Reply:
This introduced a typo, so I created a different commit